README.md

```diff
@@ -12,15 +12,15 @@ To install it, please run:
 python3 -m pip install web.py==0.62
 ```
 
-If you are still using Python 2.7, then please use web.py version 0.51
-which is intended to be our last release that supports Python 2. 
+If you are using Python 2.7, then please use web.py version 0.51
+which is our last release that supports Python 2. 
 ```
 # For Python 2.7
 python2 -m pip install web.py==0.51
 ```
 
 You can also download it from [GitHub Releases](https://github.com/webpy/webpy/releases)
```

Comment:
Now that pip contains a real dependency resolver, I no longer see any advantage in recommending this method of installation.

Reply:
Good point. We should remove it entirely as the user should never run into dependencies issues related to this.